### ISSUE TITLE
Row inside column

### ISSUE THREAD

**Issue by Skeletbossen** [linchpin/mesh#121]
First of all, thank you so much for making my new favorite Wordpress tool.

I've been looking everywhere to find a place to add a row inside a column because I'm doing a site with a design like this:

![meshissue](https://cloud.githubusercontent.com/assets/8303586/25706566/11774546-30e1-11e7-9669-6788adb2a7e0.png)

Is a design like this possible using Mesh?

Thank you.

**Comment by strarsis** [comment]
This is exactly a layout I struggled to realize with! 

I tried different page builders, but I wasn't able to achieve this kind of 
layout (rowspan/multirow) with any of these.

The content should also vertically stretch/align, 
but this is something I could control myself using custom CSS instead the stock Mesh CSS variants.

**Comment by aaronware** [comment]
@Skeletbossen This request is kinda related to #71. Not sure if @prkirby had the time to take a crack at it. You can definitely accomplish the layout you have illustrated using standard Foundation row/column markup. Simply create a 2 column section (in mesh) and in the right column use the following snippet.

`<div class="row">
<div class="small-12 medium-6 columns">
smaller left column
</div>
<div class="small-12 medium-6 columns">
smaller-right column
</div>
</div>`

In order to do the layout in Mesh from a UI standpoint that's a bigger undertaking. Mainly due to how  we're using TinyMCE along with Child/Parent posts.

**Comment by strarsis** [comment]
@aaronware: Right, the issue is rather how to make this layout editable for the backend user in an intuitive enough way. Ideally the input areas reflect the layout so the user knows immediately what to edit.

**Comment by Skeletbossen** [comment]
@aaronware Thank you, this is definitely a workaround I can use at the time being. 
I can see that adding the UI for it could be a challenge. I haven't had time to look at your code yet, but so far it has been working wonders.

Thanks again.

**Comment by prkirby** [comment]
Hey Guys, 

Unfortunately I haven't had a chance to take a crack at it. My agency is trying to move towards Craft CMS, so around the same time I created the issue, I dove into that haha. We still use wordpress, and its still on my radar to take a crack at it because we use mesh in all of our WP projects.

**Comment by strarsis** [comment]
@prkirby: Would Craft CMS offer this kind of layout out of the box? 😄

**Comment by prkirby** [comment]
@strarsis No, quite the opposite. I like craft because it imposes nothing at all. There are no prebuilt templates, there are no fields available before you make them, etc. You build everything as a standard twig template and use the fields you make, it feels a lot like developing a static site with a custom backend. DM if you want more details, this isn't really the thread for it haha.
